File: repos_estudiantes_ALVAREZ_tercera_entrega/CHINO_BLANCO_NOEMI/Generacidad/Ejercicio1/caja.py

class Caja:
    def __init__(self):
        self._contenido = None

    def guardar(self, objeto):
        self._contenido = objeto

    def obtener(self):
        return self._contenido

caja_texto = Caja()
caja_texto.guardar("cajas de abceso")

caja_numero = Caja()
caja_numero.guardar(1231)
print("Contenido de caja_texto:", caja_texto.obtener())
print("Contenido de caja_numero:", caja_numero.obtener())
print("noemi chino blanco")

File: repos_estudiantes_ALVAREZ_tercera_entrega/CHINO_BLANCO_NOEMI/Generacidad/Ejercicio1/Caja.java

public class Caja<T> {
    private T contenido;

    public void guardar(T objeto) {
        this.contenido = objeto;
    }

    public T obtener() {
        return contenido;
    }

    public static void main(String[] args) {
        Caja<String> cajaTexto = new Caja<>();
        cajaTexto.guardar("demostracion");

        Caja<Integer> cajaNumero = new Caja<>();
        cajaNumero.guardar(1224);

        System.out.println("Contenido de cajaTexto: " + cajaTexto.obtener());
        System.out.println("Contenido de cajaNumero: " + cajaNumero.obtener());
        System.out.println("noemi chino blanco");
    }
}
